jbake-core/src/test/java/org/jbake/app/AssetTest.java

```diff
@@ -152,6 +176,31 @@ public void testCopyAssetsFromContent(){
         Assert.assertTrue("Errors during asset copying", asset.getErrors().isEmpty());
     }
 
+    @Test
+    public void testIsFileAsset() {
+        File cssAsset = new File(config.getAssetFolder().getAbsolutePath() + File.separatorChar + "css" + File.separatorChar + "bootstrap.min.css");
+        Assert.assertTrue(cssAsset.exists());
+        File contentFile = new File(config.getContentFolder().getAbsolutePath() + File.separatorChar + "about.html");
+        Assert.assertTrue(contentFile.exists());
+        Asset asset = new Asset(config);
+
+        Assert.assertTrue(asset.isAssetFile(cssAsset));
+        Assert.assertFalse(asset.isAssetFile(contentFile));
+    }
+
+    /*
+    @Test
+    public void testAssetTargetFolder() throws Exception {
```

Comment:
Maybe remove that comment entirely. Or add an @Ignore with a description why it should be ignored for now.
What do you think?